### PR TITLE
IH-13 Add popup registration membership error

### PR DESCRIPTION
**Pull request by Sotatek-NgaNguyen**
### Ticket
[IH-13](https://jira.sotatek.com/browse/IH-13) 

### Attachment: Example Page
image

In case registered:
image

### CODE REVIEW

File: components/user/modal/UDialogModalError.vue [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    flex-direction: column;
+
+    img {
+      margin-bottom: 2rem;
+    }
+  }
+}
+
+@media screen and (min-width: 769px) {}
+</style>
```
Comment (Sotatek-CuongLe) [inline components/user/modal/UDialogModalError.vue:66]: add eol

File: pages/membership/modal-error.vue [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  }
+};
+</script>
+
+<style lang="scss" scoped>
+.container {
+  height: 500px;
+}
+
+</style>
```
Comment (Sotatek-CuongLe) [inline pages/membership/modal-error.vue:43]: add eol

File: pages/membership/modal-error.vue [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+import UButton from '~/components/user/common/UButton';
+import UDialogModalError from '~/components/user/modal/UDialogModalError.vue';
+
+export default {
+  name: 'ModalErrorPage',
+  components: { UButton, UDialogModalError },
+  data() {
+    return {
+      modal1: false,
```
Comment (Sotatek-CuongLe) [inline pages/membership/modal-error.vue:26]: please rename modal1 to anything more clear

File: pages/membership/modal-error.vue [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      msgError: 'Message Error Here'
```
Comment (Sotatek-CuongLe) [inline pages/membership/modal-error.vue:27]: default is empty mesage

File: components/user/modal/UDialogModalError.vue (hunk 1 above)
Comment (Sotatek-NgaNguyen) [reply components/user/modal/UDialogModalError.vue]: @Sotatek-CuongLe em update rồi ạ.

**Review by Sotatek-CuongLe — APPROVED** [review]

**Pull request merged**